src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -255,6 +266,14 @@ public void canCreateUpdateAndDeleteFares() throws IOException, SQLException, In
         assertThat(updatedFareDTO.fare_id, equalTo(updatedFareId));
         assertThat(updatedFareDTO.fare_rules[0].fare_id, equalTo(updatedFareId));
 
+        // Ensure transfers value is updated correctly.
```

Comment:
Can you expand this comment a bit to indicate that this part of the test is checking the database integrity?